torch/_inductor/utils.py

```diff
@@ -456,7 +456,7 @@ def sort_func(elem):
 RV = TypeVar("RV", covariant=True)
 
 
-class CachedMethod(Protocol, Generic[P, RV]):
+class CachedMethod(Protocol, Generic[RV, P]):
```

Comment:
You'll need to make similar change on line 469, don't you?
Also, it would be nice to add it to typing tests of sort, not to regress further